packages/chart/src/XYAxis.ts

```diff
@@ -602,6 +602,13 @@ export class XYAxis extends SVGWidget {
     @publish(26, "number", "maxPointSize")
     maxPointSize: publish<this, number>;
 
+    @publish([], "array", "Array of row indices to display values at their points")
+    showValuesFor: publish<this, number[]>;
```

Comment:
Why is this not just `showValues` similar to Columns?

Reply:
I was thinking `showValues` implies a boolean. I added the "For" as an indicator that the input expects an array of column indices. Maybe `showValuesByIndex`?